### ISSUE TITLE
TOTP copy on iOS 14

### ISSUE THREAD

**Issue by thiagoandf** [bitwarden/mobile#1020]
## Describe the Bug

The TOTP copy (both manual and auto-copy) are currently not working

## Steps To Reproduce

Manual copy:
1. Open either the App Extension or the App itself
2. Find the saved password
3. Click on "Copy TOTP"

Auto Copy
1. Let iOS auto-fill a password using bitwarden.
2. Try to paste the auto-copied TOTP

## Expected Result

The current TOTP code would be copied to the clipboard

## Actual Result

Nothing is copied to the clipboard

## Screenshots or Videos

Video: https://drive.google.com/file/d/1HFNVd5JWLq5fd20ZHWePvJ5dwUQSRmDB

## Environment

iPhone 11 Pro Max
iOS 14 Open Beta 2
2.5.0 (193)
Beta iOS, not beta App

**Comment by cscharf** [comment]
In iOS 14 Beta 2 and 3, there are no changes required for apps to access the user's clipboard, however iOS should prompt the user for permission to do so. Like a lot of iOS permissions, my assumption is that if the prompt did not happen you may need to explicitly go to your device settings and check whether there is a stored denial for this permission for the Bitwarden app, etc. OR, perhaps try updating from Beta 2 to Beta 3 to determine if there was/is a bug in the Beta in regards to the new clipboard permissions that has since been resolved. Currently there is no mention of clipboard in Apple's Beta release notes to date so not sure if that will work or not.

In short, if Bitwarden isn't granted permission to access to the clipboard by the user (or device in this instance) then clipboard access won't work within the app.

**Comment by thiagoandf** [comment]
Sorry for the delay replying. I just tested it again and it seems to be working correctly. I think it may have been a slight sync delay between registering the TOTP using the web app and it appearing in the iOS app. 
Thanks for the reply.
